service/src/test/java/uk/gov/hmcts/reform/fpl/handlers/C2UploadedEventHandlerTest.java

```diff
@@ -225,5 +224,19 @@ private AllocatedJudgeTemplateForC2 getAllocatedJudgeParametersForC2() {
 
             return allocatedJudgeTemplateForC2;
         }
+
+        private C2UploadedTemplate getC2UploadedTemplateParameters() {
+            String fileContent = new String(Base64.encodeBase64(DOCUMENT_CONTENT), ISO_8859_1);
+            JSONObject jsonFileObject = new JSONObject().put("file", fileContent);
+
+            C2UploadedTemplate uploadC2Template = new C2UploadedTemplate();
+
+            uploadC2Template.setCallout(subjectLine);
+            uploadC2Template.setRespondentLastName("Smith");
+            uploadC2Template.setCaseUrl("null/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345");
+            uploadC2Template.setDocumentLink(jsonFileObject.toMap());
```

Comment:
same here, we could use builder but this is okay too

Reply:
I think setter is fine so leaving as is